ecc-functions/owners-initiatetransfer/__init__.py

```
@@ -0,0 +1,8 @@
+import json, secrets, azure.functions as func
+
+def main(req: func.HttpRequest) -> func.HttpResponse:
+    try:
+        body = req.get_json() if req.get_body() else {}
+        return func.HttpResponse(json.dumps({"transferId": secrets.token_hex(12), "ok": True, "echo": body}), mimetype="application/json")
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)